frontend/src/js/components/devices/dialogs/DeviceConnectionDialog.tsx

```diff
@@ -86,66 +86,68 @@ const OnPremDeviceConnectionExplainer = ({ isEnterprise }) => (
-        <div className="flexbox column centered">
-          <div className="flexbox centered os-list">
-            {[raspberryPi, raspberryPi4].map((tile, index) => (
-              <img key={`tile-${index}`} src={tile} />
-            ))}
+      <div className={`margin-top-small padding-small rpi-quickstart ${classes.rpiQuickstart} ${classes.deviceSection}`}>
+        <div className="flexbox margin-bottom-small space-between">
+          <div className="flexbox centered">
+            <img src={raspberryPi} alt="rpi-logo" />
+            <Typography variant="subtitle1">Raspberry Pi quick start</Typography>
           </div>
-          <Button variant="contained" color="secondary" onClick={() => setOnDevice(true)}>
+          <Chip size="small" label="Standard" />
+        </div>
+        <Typography variant="body1">
+          A step-by-step guide for new users — connect your Raspberry Pi and get started with your first update using Mender.
+        </Typography>
+        <div className="flexbox margin-top-small">
+          <Button variant="text" color="primary" size="small" onClick={() => setOnDevice(true)} endIcon={<ArrowForwardIcon />}>
             Get started
           </Button>
         </div>
       </div>
       <div className="two-columns margin-top-small">
         <div className={`padding-small padding-bottom-none flexbox column ${classes.deviceSection}`}>
-          <div className="flexbox center-aligned">
-            <Typography variant="subtitle1" gutterBottom>
-              Use a virtual device
-            </Typography>
-            <img src={docker} className={classes.virtualLogo} />
+          <div className="flexbox space-between">
+            <div className="flexbox centered">
+              <img src={zephyr} className={classes.zephyrLogo} />
+              <Typography variant="subtitle1" gutterBottom>
+                Zephyr MCU
+              </Typography>
+            </div>
+            <Chip size="small" label="Micro" />
           </div>
-          <Typography variant="body1">
-            Don&apos;t have a Raspberry Pi?
-            <br />
-            You can use our Docker-run virtual device to go through the same tutorial.
-          </Typography>
+          <Typography variant="body1">Connect an Espressif ESP32-S3 DevKitC, or any compatible microcontroller that supports MCUBoot in Zephyr.</Typography>
           <div>
-            <Typography variant="body1" color="text.secondary">
-              If you want to evaluate our commercial components such as mender-monitor, please use a physical device instead as the virtual client does not
-              support these components at this time.
-            </Typography>
-            <Button variant="text" size="small" onClick={() => setVirtualDevice(true)}>
-              Try a virtual device
+            <Button variant="text" size="small" color="primary" endIcon={<ArrowForwardIcon />} onClick={() => setMcu(true)}>
```

Comment:
Not critical at all, but how about:
```suggestion
            <Button variant="text" size="small" endIcon={<ArrowForwardIcon />} onClick={() => setMcu(true)}>
```
here and in other places, just to limit the color adjustments if there comes another round of button color changes 🙈